clients/js/src/vaa.ts

```diff
@@ -302,8 +302,7 @@ function vaaBody(vaa: VAA<Payload | Other>) {
   return body.join("");
 }
 
-export function sign(signers: string[], vaa: VAA<Payload>): Signature[] {
-  const hash = vaaDigest(vaa);
+export function sign(signers: string[], hash: string): Signature[] {
```

Comment:
changed this fn to take the hash instead of the full VAA, updated where called elsewhere